interface/main/tabs/js/tabs_view_model.js

```diff
@@ -19,6 +19,11 @@ function tabStatus(title,url,name,closable,visible,locked)
     self.title=ko.observable(title);
     self.url=ko.observable(url);
     self.name=ko.observable(name);
+    self.loading_text=ko.observable((xl("Loading") + " " + label + "..."));
```

Comment:
This Loading is going to produce a undefined on the tab after logging in (unless I am mistaken), but I have a solution to use rather than needing to invoke my preloading workaround. Incorporate the Loading into the label when call the tabStatus. See my 2 posts below for details.

Reply:
After follow my above posts, can then change this to:
```
self.loading_text=ko.observable(label);
```